arches_lingo/src/arches_lingo/components/scheme/editor/SchemeEditor.vue

```diff
@@ -20,25 +14,45 @@ const props = defineProps<{
     activeTab: string;
     tileId?: string;
 }>();
+
+type SchemeComponent = {
+    component: SectionTypes;
+    id: string;
+    editorName: string;
+};
+
 const childRefs = ref<Array<SectionTypes>>([]);
+const currentEditor = ref<SchemeComponent>();
 const schemeComponents = [
     {
         component: SchemeLabel,
         id: "label",
-        editorTabName: $gettext("Scheme Label"),
+        editorName: $gettext("Scheme Label"),
     },
     {
         component: SchemeNamespace,
         id: "namespace",
-        editorTabName: $gettext("Scheme Namespace"),
+        editorName: $gettext("Scheme Namespace"),
     },
     {
         component: SchemeStandard,
         id: "standard",
-        editorTabName: $gettext("Scheme Standards Followed"),
+        editorName: $gettext("Scheme Standards Followed"),
     },
 ];
 
+watch(
+    props,
+    (newValue) => {
+        if (newValue) {
+            currentEditor.value = schemeComponents.find((component) => {
+                return component.id === newValue.activeTab;
```

Comment:
nit: is `activeTab` naming still needed, or will it be confusing for code maintenance in the long run?

Reply:
not needed.  purged.